### PR TITLE
Add choco package to tkn repo

### LINKED ISSUE #652: Move Chocolatey Package to More Public Area
Currently the Chocolatey package is located at https://github.com/danielhelfand/chocolatey-tektoncd-cli

At some point, it would make sense to move it to a more public location. It could either have its own repo like the [homebrew repo](https://github.com/tektoncd/homebrew-tools), or we could come up with a more central location for all the Tekton packages.

### PR DESCRIPTION
**Pull request by danielhelfand**
Closes #652 

# Changes

Added the choco package to tkn repo to help improve visibility of it. Open to any suggestions on formatting package, etc.

# Submitter Checklist

- [ ] Includes [tests](https://github.com/tektoncd/community/blob/master/standards.md#principles) (if functionality changed/added)
- [x] Run the code checkers with `make check`
- [ ] Regenerate the manpages, docs and go formatting with `make generated`
- [x] Commit messages follow [commit message best practices](https://github.com/tektoncd/community/blob/master/standards.md#commit-messages)

# Release Notes

```release-note
NONE
```

### CODE REVIEW

**Comment by bobcatfish** [comment]
/cc @tektoncd/cli-collaborators 

I noticed a review was requested for me on this b/c I'm in cli-maintainers - so i went to remove myself from that b/c i probably shouldn't be reviewing CLI stuff XD - but then i thought, maybe what we actually wanted here was to request a review from cli-collaborators

File: choco/tools/chocolateyinstall.ps1 [hunk 1]
```diff
@@ -0,0 +1,15 @@
+﻿$ErrorActionPreference = 'Stop'; 
+$toolsDir    = "$(Split-Path -parent $MyInvocation.MyCommand.Definition)"
+$packageName = 'tektoncd-cli'
+$url64       = 'https://github.com/tektoncd/cli/releases/download/v0.18.0/tkn_0.18.0_Windows_x86_64.zip'
```
Comment (pradeepitm12) [inline choco/tools/chocolateyinstall.ps1:4]: May be we would like to update the version here.

**Comment by pradeepitm12** [comment]
Can someone help to test/verify this?

File: choco/tools/chocolateyinstall.ps1 (hunk 1 above)
Comment (piyush-garg) [reply choco/tools/chocolateyinstall.ps1:4]: Its a placeholder, you can use it for any version from what i understood

**Pull request merged**